health/micro-ui/web/packages/modules/campaign-manager/src/pages/employee/NewAppConfiguration/components/SelectionCard.js

```
@@ -0,0 +1,46 @@
+import React from "react";
+import { Loader, SelectionTag } from "@egovernments/digit-ui-components";
+
+const SelectionCard = ({ field, t }) => {
+  const isMdmsEnabled = !!field?.isMdms && !!field?.schemaCode;
+
+  const { isLoading, data } = Digit?.Hooks.useCustomMDMS(
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)